views/email_template/plain_text/UserInviteSelf.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+
+use yii\helpers\Html;
+
+/* @var $token integer */
+
+?>
+<?= mb_strtoupper(Yii::t('UserModule.base', 'Welcome to %appName%', ['%appName%' => Yii::$app->name])) ?>
+
+
+<?= Yii::t('UserModule.base',
+    'Welcome to %appName%. Please enter the code below to proceed with your registration.',
```

Comment:
is this a text to be rendered in the email?

Reply:
Hi @donni106 
Yes

Reply:
done with 6b8c2b1af387e1ed1df45332001df264557139bd